core/src/components/header/header.tsx

```diff
@@ -46,6 +46,12 @@ export class Header implements ComponentInterface {
    */
   @Prop() collapse?: 'condense' | 'fade';
 
+  /**
+   * If `true`, the header will have a line at the bottom.
+   * TODO(ROU-10855): add support for this prop on ios/md themes
+   */
+  @Prop({ reflect: true }) divider = false;
```

Comment:
Do we need this as a property? We have support for a border in the other themes by applying it by default in CSS. If a user wants to remove it, they can add the `ion-no-border` CSS class to `ion-header`. Here is how it looks for `ios` (it only applies the border to the last toolbar in a header):

```scss
// header.ios.scss

.header-ios ion-toolbar:last-of-type {
  --border-width: 0 0 #{$hairlines-width};
}

.header-ios.ion-no-border ion-toolbar:last-of-type {
  --border-width: 0;
}
```

Reply:
I understand we can do it with CSS + utilities, but it was followed what was defined by the UX/UI team to have this property. And as this has small impact I think its a good opportunity to make the experience more similar between our different contexts.

But we can bring this to the rest of the team, if you feel this isn't really the best approach!